keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -1028,6 +1029,17 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
     This will not happen anymore.
   - The CRI implementation may scrape the metrics less efficiently than cAdvisor currently does. This should be measured and evaluated as a requirement of Beta.
 
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+It is possible the CRI implementation could use more memory or CPU for the metrics gathering than cAdvisor does. In this case,
+one of those resources could be in jepordy of being exhausted. However, it is a explicit goal of beta criteria (and graduation generally)
+to prevent such regressions, so this feature will not be on by default if this is possible.
```

Comment:
It was my thought that it would happen in this order:
- verify performant, accurate, safe
- mark cadvisor deprecated
- mark KEP beta

this way, we have a chance to deprecate cadvisor before the KEP is graudated, to give users a chance to prepare for the switch